common/src/main/java/tachyon/security/login/TachyonJaasConfiguration.java

```diff
@@ -48,9 +48,9 @@
   // private static final AppConfigurationEntry KERBEROS_LOGIN = ...
 
   /**
-   * In SIMPLE mode, JAAS use the OS specific login module to fetch the OS user,
-   * and then use the {@link tachyon.security.TachyonLoginModule} to convert it to a Tachyon user
-   * represented by {@link tachyon.security.User}
+   * In SIMPLE mode, JAAS use the OS specific login module to fetch the OS user, and then use the
```

Comment:
Since #1292 , we also allow a customized username to be specified. So when you resolves the conflicts of this PR with the security branch, can you also update the description? Thanks.